### PR TITLE
renaming of the stopping criterion 

### PR DESCRIPTION
**Pull request by omursahin**

### CODE REVIEW

File: core/src/main/kotlin/org/evomaster/core/docs/ConfigToMarkdown.kt [hunk 1]
```diff
@@ -156,7 +157,7 @@ object ConfigToMarkdown {
         val typeName = if(type.isEnum){
             "Enum"
         } else {
-            type.simpleName.capitalize()
+            type.simpleName.replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.ENGLISH) else it.toString() }
```
Comment (arcuri82) [inline core/src/main/kotlin/org/evomaster/core/docs/ConfigToMarkdown.kt:160]: why this change?

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: core/src/main/kotlin/org/evomaster/core/docs/ConfigToMarkdown.kt (hunk 1 above)
Comment (omursahin) [reply core/src/main/kotlin/org/evomaster/core/docs/ConfigToMarkdown.kt:160]: Since I changed the parameter name, I had to generate Markdown again. When I regenerated Markdown, the tests ran correctly on my computer but gave an error in the CI process (https://github.com/WebFuzzing/EvoMaster/actions/runs/11477931721/job/31942213194). Later, I realized that since it capitalized the "ı" character as "İ", the "Int" type was written as "İnt" in the file. The tests pass/fail depending on the computer's default language. With this change, I aimed to remove the dependency on the language. 
Btw, the capitalize function appeared to be deprecated so the IDE suggested me to use the replaceFirstChar function instead.

image

Comment (omursahin) [reply core/src/main/kotlin/org/evomaster/core/docs/ConfigToMarkdown.kt]: I think this time It is OK.

**Review by arcuri82 — APPROVED** [review]
